package.json

```diff
@@ -3,17 +3,17 @@
   "version": "1.0.0",
   "description": "Simorgh",
   "scripts": {
-    "amp:validate": "wait-on http://localhost:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost:7080/news/articles/c6v11qzyv8po.amp",
-    "bbcA11y": "wait-on -t 20000 http://localhost:7080/news/articles/c6v11qzyv8po && bbc-a11y",
+    "amp:validate": "wait-on http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp && amphtml-validator http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po.amp",
+    "bbcA11y": "wait-on -t 20000 http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po && bbc-a11y",
     "build:ci": "export NODE_ENV=production && rm -rf build && npm run build:test && npm run build:live",
     "build": "rm -rf build && cp envConfig/local.env .env && NODE_ENV=production webpack",
     "build:storybook": "build-storybook -c .storybook -o storybook_dist",
     "build:test": "cp envConfig/test.env .env && NODE_ENV=production webpack",
     "build:live": "cp envConfig/live.env .env && NODE_ENV=production webpack",
-    "cypress": "cypress run",
-    "cypress:interactive": "cypress open",
-    "cypress:storybook": "wait-on -t 120000 http://localhost:9001 && cypress run --project ./.storybook/.",
-    "cypress:storybook:interactive": "cypress open --project ./.storybook/.",
+    "cypress": "export NO_PROXY=localhost,localhost.bbc.com && cypress run",
```

Comment:
I can't find a cleaner way to do this - doesn't seem possible to set proxy-related environment variables using any of these methods: https://docs.cypress.io/guides/guides/environment-variables.html#Setting